CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/lib/Alias.kt

```diff
@@ -0,0 +1,3 @@
+package co.nimblehq.coroutine.lib
+
+typealias IsLoading = Boolean
```

Comment:
Can we rename this class to `TypeAlias` instead? What do you think? @chornerman

Reply:
Resolved, thank you 🙏🏼